dot/parachain/mocks_generate_test.go

```diff
@@ -4,4 +4,4 @@
 package parachain
 
 //go:generate mockgen -destination=mocks_test.go -package=$GOPACKAGE . PoVRequestor
-//go:generate mockgen -destination=mocks_runtime_test.go -package $GOPACKAGE github.com/ChainSafe/gossamer/dot/parachain/runtime RuntimeInstance
+//go:generate mockgen -destination=runtime_mock.go -package $GOPACKAGE github.com/ChainSafe/gossamer/dot/parachain/runtime RuntimeInstance
```

Comment:
The reason for not using the `_test` suffix was so that It can be accessed from another package. It doesn't export them otherwise. Do you know any other way to use it?

Reply:
Ah, that makes sense. Thank you!

Reply:
I've reverted the changes and have kept the mocks in the dispute package itself!